.github/workflows/ci.yml

```diff
@@ -76,6 +76,29 @@ jobs:
           architecture: "x64"
       - run: pip install "ray>=1.0.0"
 
+  lint-pylint:
+    needs: prepare-cache
+    runs-on: ubuntu-latest
+    name: run pylint
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - name: Cache pip
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-python-3.7-pip-${{ github.run_id }}-${{ hashFiles('environment.yml') }}
+      - uses: goanpeca/setup-miniconda@v1.6.0
+        with:
+          activate-environment: modin
+          environment-file: environment.yml
+          python-version: 3.7
+          channel-priority: strict
+          use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
```

Comment:
Is it mandatory for the `pylint` to create a complete dev environment?

Reply:
I'm removing `pylint` section for now, it should be a separate PR.